nucypher/blockchain/eth/domains.py

```diff
@@ -0,0 +1,70 @@
+from enum import Enum
+from typing import NamedTuple
+
+
+class ChainInfo(NamedTuple):
+    id: int
+    name: str
+
+
+class EthChain(ChainInfo, Enum):
+    MAINNET = (1, "mainnet")
+    GOERLI = (5, "goerli")
+    SEPOLIA = (11155111, "sepolia")
+    TESTERCHAIN = (131277322940537, "eth-tester")
```

Comment:
Can `TESTERCHAIN` be relocated to mocking / testing facilities?

Reply:
It can be, and I've now done that. My initial hesitation for doing that was that type wise it's not technically correct - we would only be able to create a domain with `ChainInfo` objects, and not `EthChain`/ `PolygonChain` objects because they are enums. 

In any case, the move works because of the `ChainInfo` base class.

Reply:
✔️